lib/elixir/src/elixir_expand.erl

```diff
@@ -870,6 +876,8 @@ format_error({missing_option, Construct, Opts}) when is_list(Opts) ->
   io_lib:format("missing ~ts option in \"~ts\"", [string:join(StringOpts, "/"), Construct]);
 format_error({invalid_args, Construct}) ->
   io_lib:format("invalid arguments for \"~ts\"", [Construct]);
+format_error({for_invalid_uniq, Value}) ->
+  io_lib:format(":uniq option for comprehension only accept boolean, got: ~ts", ['Elixir.Macro':to_string(Value)]);
```

Comment:
The error wording could use some pluralization: ":uniq option for comprehensions only accepts a boolean"